sql/npc_list.sql

```diff
@@ -20496,8 +20496,8 @@ INSERT INTO `npc_list` VALUES (17494713,'Layton','Layton',105,382.679,-39.999,3.
 INSERT INTO `npc_list` VALUES (17494714,'Erik','Erik',125,258.643,-33.249,99.901,9,40,40,0,0,0,0,27,0x0100060102100220023002400250486000700000,32,'WOTG',1);
 INSERT INTO `npc_list` VALUES (17494715,'Theodor','Theodor',132,258.693,-33.250,-60.032,1,40,40,0,0,0,0,27,0x01000C0102100220023002400250616000700000,32,'WOTG',1);
 INSERT INTO `npc_list` VALUES (17494716,'RedAxe','Red Axe',130,304.628,-27.500,13.955,26,40,40,0,0,0,0,27,0x01000A08021002200230024002504F6100700000,32,'WOTG',1);
-INSERT INTO `npc_list` VALUES (17494717,'Gravestone','Gravestone',0,254.428,-32.999,20.001,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',1);
-INSERT INTO `npc_list` VALUES (17494718,'Sarcophagus','Sarcophagus',0,336.594,-33.500,-56.728,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',1);
+INSERT INTO `npc_list` VALUES (17494717,'Gravestone_map_quest','Gravestone',0,254.428,-32.999,20.001,1,40,40,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'WOTG',1);
```

Comment:
Surprisingly, Gravestone appears to be unique in this zone, so the suffix for it isn't necessary (to others: the second Sarcophagus is 17494754).

Reply:
@ibm2431  how does the map know which one to choose? I posted in development and I kinda understood that the names were unique. Would love insight to get a better understanding 😸️

Reply:
It [builds an execution path based on the current zone and the triggered NPC's name](https://github.com/DarkstarProject/darkstar/blob/6b9f3d77eb2201b923160c50d13379d9f3b1ee8a/src/map/lua/luautils.cpp#L1440-L1442).

Reply:
Reverted the name back to the original. Thank you 😸️